scripts/py_matter_yamltests/matter_yamltests/constraints.py

```diff
@@ -75,6 +82,8 @@ def check_response(self, value) -> bool:
             success = True
         elif self._type == 'char_string' and type(value) is str:
             success = True
+        elif self._type == 'long_char_string' and type(value) is str:
```

Comment:
Do you not need `long_octet_string` too?

Reply:
While there are no constrain type checking for this type I have added it in